src/main/java/nextstep/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class DeleteHistories {
+
+    private final List<DeleteHistory> histories;
+
+    public DeleteHistories() {
+        this.histories = new ArrayList<>();
+    }
+
+    public DeleteHistories(List<DeleteHistory> histories) {
+        this.histories = histories;
+    }
```

Comment:
이러한 경우 생성자 연쇄를 사용하시면 응집도 있는 코드를 작성하기 용이합니다.

```suggestion
    public DeleteHistories() {
        this(new ArrayList<>());
    }

    public DeleteHistories(List<DeleteHistory> histories) {
        this.histories = histories;
    }
```